applications/nexus/nex2ecl.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+   Copyright (C) 2017  Statoil ASA, Norway.
+   The file 'nex2ecl' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <string>
+
+#include <iostream>
+
+#include <ert/nexus/nexus_plot.hpp>
+
+
+void create_ecl_sum(char *file_path) {
+    std::string filename( file_path );
+    auto ending = filename.rfind( ".plt" );
+    if (ending == std::string::npos) {
+        filename += ".plt";
+    }
+
+    nex::NexusPlot plt = nex::load( filename );
+
+    std::string basename( filename );
+    basename.erase(basename.find_last_of( '.' ), basename.size());
```

Comment:
For Eclipse output the basename should be `ALL_CAPS`. In addition it is the question of what to with a possible directory part - as I see it there are two possibilities:

1. The ECLIPSE output will be generated in cwd, irrespective of where the nexus case is located. In that case it is important to strip off the leading path component.
2. The ECLIPSE output will be generated in the same location as the nexus case is located, in that case it is important that *only* the base name is UPPERCASED - and not the directory part.

Reply:
Now option 2 is implemented, output file will be given the same name as input, but in upper case. It will be generated in the same folder as of now, I can add an option to output in spesified folder with syntax like: `nex2ecl input.plt -o /output/directory/` if desired.